src/Radicle/Internal/PrimFns.hs

```diff
@@ -483,11 +480,12 @@ purePrimFns = fromList $ allDocs $
       )
     , ( "member?"
       , "Given `v` and structure `s`, checks if `x` exists in `s`. The structure `s`\
-        \ may be a list, vector or dict. If it is a list or a vector, it checks if `v`\
+        \ may be a sequence or dict. If it is a sequence, it checks if `v`\
```

Comment:
Wouldn’t it be clearer to have to have a `elem?` prinfn and a `member?` function that only works for dicts?